CorsixTH/Lua/app.lua

```diff
@@ -1696,6 +1713,7 @@ function App:load(filepath)
   if self.world then
     self:worldExited()
   end
+  self.save_name = filepath:match(".*" .. pathsep .. "(.+)%.sav$")
```

Comment:
When player saves a game, probably useful to refresh self.save_name too.

Does `Continue Game` from Main Menu also check the autosaves folder? ... I don't think it does but if it does it could be an issue.

Reply:
It does, so continuing a named save will be logged in files named for the autosave. That is possible to fix for the future, this save_name can be in world, but it could be confusing for existing games. I'm thinking about taking the save name out of the filename, so at least old gamelogs aren't overwritten.